google/cloud/storage/internal/access_control_common_parser.cc

```diff
@@ -36,10 +36,12 @@ Status AccessControlCommonParser::FromJson(AccessControlCommon& result,
   result.self_link_ = json.value("selfLink", "");
   if (json.count("projectTeam") != 0) {
     auto tmp = json["projectTeam"];
-    ProjectTeam p;
-    p.project_number = tmp.value("projectNumber", "");
-    p.team = tmp.value("team", "");
-    result.project_team_ = std::move(p);
+    if (!tmp.is_null()) {
```

Comment:
Could you change this to `if (tmp.is_null()) return Status{};` ?

Reply:
sure thing, I changed it in https://github.com/googleapis/google-cloud-cpp/pull/8446/commits/739c7f0e3af09ee680c62ab9e4565a224e0966c2